ALGORITHMS.md

```diff
@@ -72,23 +72,23 @@ As standardization for these algorithms within TLS is not done, all TLS code poi
 | p384_dilithium3 | 0xfea4 |Yes| OQS_CODEPOINT_P384_DILITHIUM3
 | dilithium5 | 0xfea5 |Yes| OQS_CODEPOINT_DILITHIUM5
 | p521_dilithium5 | 0xfea6 |Yes| OQS_CODEPOINT_P521_DILITHIUM5
-| mldsa44 | 0xfed0 |Yes| OQS_CODEPOINT_MLDSA44
-| p256_mldsa44 | 0xfed3 |Yes| OQS_CODEPOINT_P256_MLDSA44
-| rsa3072_mldsa44 | 0xfed4 |Yes| OQS_CODEPOINT_RSA3072_MLDSA44
+| mldsa44 | 0xff06 |Yes| OQS_CODEPOINT_MLDSA44
+| p256_mldsa44 | 0xff09 |Yes| OQS_CODEPOINT_P256_MLDSA44
+| rsa3072_mldsa44 | 0xff0a |Yes| OQS_CODEPOINT_RSA3072_MLDSA44
```

Comment:
given that the hybrids don't use the OIDs from [draft-ietf-lamps-pq-composite-sigs-03](https://datatracker.ietf.org/doc/html/draft-ietf-lamps-pq-composite-sigs-03), I'm assuming that those are not the LAMPS hybrids just yet?

if that's not the case, those algorithms should use IDs from [draft-reddy-tls-composite-mldsa-01](https://www.ietf.org/id/draft-reddy-tls-composite-mldsa-01.html)

Reply:
Thanks for the pointer, @tomato42! I’ve updated the IDs based on draft-reddy-tls-composite-mldsa-01, except for the brainpool composites, which don’t seem to have an ID defined in the draft. Since I’m not fully familiar with the details of the composite implementation, @feventura, could you please confirm if these IDs align with the implementation?

Reply:
The Brainpool composites are explicitly excluded from the TLS draft, so yes, they still need to use custom codepoints.

Reply:
the thing is that the lamps draft is not just concatenating two signatures: it uses the two algorithms to sign a separate synthetic message: https://datatracker.ietf.org/doc/html/draft-ietf-lamps-pq-composite-sigs-03#section-4.2.1

Reply:
I honestly don't know... if it passes interop, then yes :slightly_smiling_face:

Reply:
Tagging @praveksharma : How's the IETF hackathon interop test image/run coming along?

Reply:
Thanks for the confirmation via meeting that interop is OK with quite a few other implementations in the IETF hackathon. Considering the issue resolved then.